autowrap/CodeGenerator.py

```diff
@@ -191,6 +191,7 @@ def setup_cimport_paths(self):
 
         pxd_dirs = set()
         for inst in self.all_classes + self.all_enums + self.all_functions + self.all_typedefs:
+            # Could this not simply be `for inst in self.all_resolved:` ?
```

Comment:
I'll remove this comment but I just wanted to make sure there was a note here. If I can change this to `for inst in self.all_resolved`, I think it would read better (never mind if those two lines are not actually identical).

Reply:
I think you are correct. Should be equivalent.